RecoBTag/CTagging/python/positiveCharmTagJetTags_cfi.py

```diff
@@ -0,0 +1,16 @@
+import FWCore.ParameterSet.Config as cms
+
+pfPositiveCombinedCvsLJetTags = cms.EDProducer(
```

Comment:
please match the file name closer to the module instance definitions.
Also, multiple module definitions better be done in cff files